### PR TITLE
Update to v4.0.0

### LINKED ISSUE #3: Update to v4.0.0

### PR DESCRIPTION
**Pull request by paolostivanin**
- fix #3 
- add patch to compile against gcc 8
- update Gnome stack to 3.28
- use tarball instead of latest git commit
- add polkit

ping @nedrichards :)

### CODE REVIEW

**Comment by LeandroStanger** [comment]
@paolostivanin upgrade to gnome stack> 3.28

**Comment by paolostivanin** [comment]
I tried but with gnome > 3.26 gtkglext doesn't compile. It probably needs to be patched I guess. Tomorrow I will look it deeper into it.

**Comment by paolostivanin** [comment]
done, updated also Gnome :)

File: com.anydesk.Anydesk.json [hunk 1]
```diff
@@ -47,11 +47,25 @@
...
+              "make",
+              "make install"
+            ],
             "sources": [
                 {
-                    "type": "git",
-                    "url": "https://gitlab.gnome.org/Archive/gtkglext.git",
-                    "commit": "04131c5f69070b2f194f02edd45b0622ca03793c"
+                    "type": "archive",
+                    "url": "http://downloads.sourceforge.net/sourceforge/gtkglext/gtkglext-1.2.0.tar.bz2",
```
Comment (nedrichards) [inline com.anydesk.Anydesk.json]: We don't recommend sourcefourge hosting where possible due to unreliability. Could you try https://download.gnome.org/sources/gtkglext/1.2/gtkglext-1.2.0.tar.bz2 (I may fix this myself in manual testing)

**Comment by nedrichards** [comment]
bot, build

**Comment by LeandroStanger** [comment]
Update the va icon in icons/scalable/apps

File: com.anydesk.Anydesk.json [hunk 2]
```diff
@@ -70,7 +84,7 @@
                 }
             ]
         },
-        
+
```
Comment (nedrichards) [inline com.anydesk.Anydesk.json]: stray newline, we should remove.

**Comment by paolostivanin** [comment]
@nedrichards no, sorry. I tested only the compilation I didn't run it. But yes, true, it crashes.

I'll look into it tonight/tomorrow

**Comment by nedrichards** [comment]
bot, build com.anydesk.Anydesk

**Comment by nedrichards** [comment]
@paolostivanin right, yeah - for extra-data apps you *have* to test the install step since the actual app isn't actually there at compile time.

**Comment by nedrichards** [comment]
This works slightly better than the previous version in local testing. merging.

**Pull request merged**